projects/1.19/assets/heat-and-climate/dcs_climate/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+  "item.dcs_climate.food.pack_coconut_milk":"椰奶盒",
+  "item.dcs_climate.food.pack_cream":"奶油盒",
+  "item.dcs_climate.food.pack_whip":"植物奶油盒",
+  "item.dcs_climate.food.pack_honey":"蜂蜜罐",
+  "item.dcs_climate.food.pack_syrup":"糖浆罐",
+  "item.dcs_climate.food.pack_plant_oil":"植物油罐",
+  "item.dcs_climate.food.pack_water":"水瓶",
+  "item.dcs_climate.food.pack_empty":"空纸盒",
+
+  "item.dcs_climate.food.stick_sorghum":"高粱杆",
+  "item.dcs_climate.food.food_palm_flower":"棕榈花",
+  "item.dcs_climate.food.food_vine":"藤条",
+  "item.dcs_climate.food.fiber_plant":"植物纤维",
+  "item.dcs_climate.food.fiber_wood":"树皮纤维",
+  "item.dcs_climate.food.fiber_beeswax":"蜜蜡",
+  "item.dcs_climate.food.string_plant":"植物纤维绳",
+  "item.dcs_climate.food.string_tree":"树皮绳",
+  "item.dcs_climate.food.string_wool":"毛线",
+  "item.dcs_climate.food.string_cotton":"棉线",
+
+  "item.dcs_climate.food.dust_defatted_soy":"脱脂大豆",
+  "item.dcs_climate.food.dust_press_cake":"油渣",
+
+  "item.dcs_climate.food.animalfeed_hay":"干草",
+  "item.dcs_climate.food.animalfeed_straw":"稻草",
+  "item.dcs_climate.food.animalfeed_compound":"混合饲料",
+
+  "item.dcs_climate.food.fertilizer_mixed":"复合肥",
+
+  "item.dcs_climate.food.bread_round_raw":"面团",
+  "item.dcs_climate.food.bread_round_baked":"圆面包",
+  "item.dcs_climate.food.bread_square_raw":"方面团",
+  "item.dcs_climate.food.bread_square_baked":"吐司面包",
+  "item.dcs_climate.food.bread_nuts_raw":"坚果果干面团",
+  "item.dcs_climate.food.bread_nuts_baked":"坚果果干面包",
+  "item.dcs_climate.food.bread_cinnamon_raw":"肉桂卷面团",
+  "item.dcs_climate.food.bread_cinnamon_baked":"肉桂卷",
+  "item.dcs_climate.food.bread_anko_raw":"豆沙面团",
+  "item.dcs_climate.food.bread_anko_baked":"豆沙包",
+  "item.dcs_climate.food.bread_cream_raw":"奶油夹心面团",
+  "item.dcs_climate.food.bread_cream_baked":"奶油夹心面包",
+  "item.dcs_climate.food.bread_sausage_raw":"香肠面包面团",
+  "item.dcs_climate.food.bread_sausage_baked":"香肠面包",
+
+  "item.dcs_climate.food.sandwich_fruit":"果酱三明治",
+  "item.dcs_climate.food.sandwich_marmalade":"橘子三明治",
+  "item.dcs_climate.food.sandwich_egg":"鸡蛋三明治",
+  "item.dcs_climate.food.sandwich_salad":"沙拉三明治",
+  "item.dcs_climate.food.sandwich_salmon":"三文鱼三明治",
+
+  "item.dcs_climate.food.porridge_simple":"麦粥",
+  "item.dcs_climate.food.porridge_milk":"牛奶麦粥",
+  "item.dcs_climate.food.muesli":"牛奶麦片",
+
+  "item.dcs_climate.food.stick_beef_raw":"生牛肉串",
+  "item.dcs_climate.food.stick_beef_cooked":"烤牛肉串",
+  "item.dcs_climate.food.stick_pork_raw":"生猪肉串",
+  "item.dcs_climate.food.stick_pork_cooked":"烤猪肉串",
+  "item.dcs_climate.food.stick_mutton_raw":"生羊肉串",
+  "item.dcs_climate.food.stick_mutton_cooked":"烤羊肉串",
+  "item.dcs_climate.food.stick_chicken_raw":"生鸡肉串",
+  "item.dcs_climate.food.stick_chicken_cooked":"烤鸡肉串",
+  "item.dcs_climate.food.stick_offal_raw":"生内脏串",
+  "item.dcs_climate.food.stick_offal_cooked":"烤内脏串",
+
+  "item.dcs_climate.food.plate_beef_raw":"生牛排",
+  "item.dcs_climate.food.plate_beef_cooked":"烤牛排",
+  "item.dcs_climate.food.plate_meat_raw":"生肉排",

```

```suggestion
  "item.dcs_climate.food.plate_meat_raw":"生切肉排",
```
原文`生のカットステーキプレート`，至少要把`カット`翻译出来吧